BetterHolidays/__init__.py

```
@@ -0,0 +1,5 @@
+from .days import Day, Holiday, TradingDay, PartialTradingDay
```

Ruff (0.8.2)

`.days.Day` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)